pkg/cache/cache.go

```diff
@@ -183,6 +183,12 @@ func (c *ClusterQueue) updateWorkloadUsage(wi *workload.Info, m int64) {
 		}
 	}
 }
+func (c *Cache) ClusterQueueExists(name string) bool {
```

Comment:
if this is about queueing, it should be in the queue manager, which also has a ClusterQueue representation.